packages/terra-consumer-nav/src/components/nav-help/NavHelp.scss

```diff
@@ -48,6 +47,10 @@
     }
   }
 
+  .help-item-border {
```

Comment:
class names should describe what a thing is, not what it looks like. From the code it looks like this is for n+1 help item... which I'd ask if we can use a pseudo selector to do that instead of an additional class name